cloudzero/README.md

```diff
@@ -2,45 +2,44 @@
 
 ## Overview
 
-### Overview
-CloudZero helps engineering teams build cost-effective software. Through its innovative platform, CloudZero allocates 100% of customers' cloud, PaaS, and SaaS spend - regardless of their tagging quality - and presents it in a single unified view. It then combines hourly spend data with business- and system-level telemetry to put cloud data in a business context, through precise unit cost metrics like cost per customer, per product, per feature, per team, and more. CloudZero's AI-powered anomaly detection alerts engineers to abnormal spend events, pointing them directly to the affected infrastructure, and promoting engineering engagement in cloud cost management.
+CloudZero helps engineering teams build cost-effective software. Through its innovative platform, CloudZero allocates 100% of customers' cloud, PaaS, and SaaS spend—regardless of their tagging quality—and presents it in a single unified view. It then combines hourly spend data with business- and system-level telemetry to put cloud data in a business context through precise unit cost metrics like cost per customer, per product, per feature, per team, and more. CloudZero's AI-powered anomaly detection alerts engineers to abnormal spend events by pointing them directly to the affected infrastructure, and promotes engineering engagement in cloud cost management.
```

Comment:
Some optional, very minor changes.

I see that you specifically removed the spaces around the dashes separating {spend, regardless} and {quality, and}. I'm gonna assume that was intentional; my brain wants them back but if there's a reason for that, is good.

Maybe consider "through precise unit cost metrics: per customer, per product, per feature..." or somesuch to avoid doubling up on the word "cost" too closely. (Granted, if you're gonna double up on one word at CZ, "cost" is a sensible one, but).

Unabashed grammar sticklers would want you to remove the comma between "infrastructure" and "and". (I want there to be one there too and put commas in 2-tuples myself all the time, though, so this isn't a request so much as an observation.)

Reply:
I agree with your suggestions, but in an effort to expedite the release of the integration, I took the suggestions from Datadog wholesale. We can follow up another round of changes. I'm inclined to go forward with this for now.